packages/playwright-core/src/server/chromium/crNetworkManager.ts

```diff
@@ -508,6 +514,7 @@ class InterceptableRequest {
   readonly _timestamp: number;
   readonly _wallTime: number;
   readonly _route: RouteImpl | null;
+  readonly _latestRoute: RouteImpl | undefined;
```

Comment:
Can we walk the redirect chain on the call site instead? If not, let's call this `originalRequestRoute` since we know it only exists on the first (non-redirect) request.

Reply:
We do not store reference to the previous request in InterceptableRequest, neither do we store a reference from Request to InterceptableRequest.